engine/crates/engine-v2/src/response/write/deserialize/enum.rs

```diff
@@ -0,0 +1,30 @@
+use schema::EnumDefinitionId;
+use serde::{de::DeserializeSeed, Deserialize};
+use walker::Walk as _;
+
+use crate::response::ResponseValue;
+
+use super::SeedContext;
+
+pub(crate) struct EnumTypeSeed<'parent, 'ctx>(pub &'parent SeedContext<'ctx>, pub EnumDefinitionId);
```

Comment:
Hum realizing `ScalarTypeSeed` isn't a great name , would have used `EnumValueSeed` here, but it's really just a nitpick

Reply:
pushed